kuma/wiki/jinja2/wiki/document.html

```diff
@@ -332,6 +332,25 @@
                 {% else %}
                   {{ _("This article doesn't have approved content yet.") }}
                 {% endif %}
+
+                <aside class="helpful-survey hidden" id="helpful-survey">
+                  <p>{{ _('Was this article helpful?') }}</p>
+
+                  <div class="helpful-survey-content">
+                    <div class="helpful-survey-buttons">
+                      <button class="helpful-survey-vote helpful-survey-yes">
```

Comment:
Buttons 😻 . Should have type="button" so IE doesn't treat them as submit buttons.